hydromt_wflow/wflow.py

```diff
@@ -1584,6 +1585,189 @@ def setup_temp_pet_forcing(
+
+        * **rootzone_storage_{forcing}_{RP}** map: rootzone storage capacity [mm of water] estimated from hydroclimatic data {forcing: obs, cc_hist or cc_fut} for different return periods RP.
+        * **RootingDepth_{forcing}_{RP}** map: rooting depth [mm of the soil column] estimated from hydroclimatic data {forcing: obs, cc_hist or cc_fut} for different return periods RP. The translation to RootingDepth is done by dividing the rootzone_storage by (thetaS - thetaR). 
+        * **rootzone_storage_{forcing}_{RP}** geom: polygons of rootzone storage capacity [mm of water] for each catchment estimated before filling the missings with data from downstream catchments.
+
+
+        Parameters
+        ----------
+        run_fn : str, Path, xr.Dataset
+            Geodataset with streamflow timeseries (m3/s) per x,y location. 
+            The geodataset expects the coordinate names "index" (for each station id), 
+            "x" (for x coord) and "y" for y coord. 
+        forcing_obs_fn : str, Path, xr.Dataset
+            Gridded timeseries with the observed forcing [mm/timestep]. 
+            Expects to have variables "precip" and "pet".
+        forcing_cc_hist_fn : str, Path, xr.Dataset, optional
+            Gridded timeseries with the simulated historical forcing [mm/timestep], based on a climate
+            model. Expects to have variables "precip" and "pet". The default is None.
+        forcing_cc_fut_fn : str, optional
+            Gridded timeseries with the simulated climate forcing [mm/timestep], based on a
+            climate model. Expects to have variables "precip" and "pet". 
+            The default is None.
+        chunksize : int, optional
+            Chunksize on time dimension for processing data (not for saving to 
+            disk!). The default is 100.
+        return_period : list, optional
+            List with one or more values indiciating the return period(s) (in 
+            years) for wich the rootzone storage depth should be calculated. The
+            default is [2,3,5,10,15,20,25,50,60,100] years.
+        Imax : float, optional
+            The maximum interception storage capacity [mm]. The default is 2.0 mm.
+        start_hydro_year : str, optional
+            The start month (abreviated to the first three letters of the month,
+            starting with a capital letter) of the hydrological year. The 
+            default is 'Sep'.
+        start_field_capacity : str, optional
+            The end of the wet season / commencement of dry season. This is the
+            moment when the soil is at field capacity, i.e. there is no storage
+            deficit yet. The default is 'Apr'. 
+        LAI : bool, optional
+            Determine whether the LAI will be used to determine Imax. The
+            default is False.
+            If set to True, requires to have run setup_laimaps. 
+        rooting_depth : bool, optional
+            Determines whether the rooting depth (rootzone storage / 
```

Comment:
In the docstrings you say this method sets the RootingDepth but here you  provide an optional True/False flag for it?

Reply:
very good point, I have changed this, now RootingDepth is no longer an option, but having rootzone_storage in the staticmaps is an option, see comment below